Adds content for the overview section

JackieBinya opened the pull request:
## Fixes
<!-- If PR doesn't fully resolve the issue, replace 'Fixes' below with 'Related to'. -->
<!-- If there is no issue being resolved, open one before creating this pull request. -->
Fixes #[issue number] by @[issue author]

## Description
<!-- Concisely describe what the pull request does. -->
- I have migrated the draft content for the Overview Section from Google Docs to the project repository. I tweaked the font color of the links to match that of [Creative Commons Open Source Blog](https://opensource.creativecommons.org/community/write-a-blog-post/). I would love feedback to confirm if everything is **OK**.
@hugosolar I encountered several blockers that prevented me from completing the task, please find them listed below:
1. You had said I should retrieve the screenshot from https://stage.creativecommons.org/ and I am failing to access the site as it requires authentication.
2. I lost the link you had provided with information about the licensing and reuse of CC software. The section with the missing links is captured in the screenshot below. I thought I had included the link in the draft content on Google Docs, only to find out that I hadn't, it must have had been in the chats I marked as resolved.
![Licensing Details](https://user-images.githubusercontent.com/50267279/93448983-7d781b80-f8d4-11ea-9b7c-948349e776d6.png)

## Tests
<!-- Give steps for the reviewer to verify that this PR fixes the problem; or delete the section entirely. -->
**Ensure that the [Gridsome CLI]( https://gridsome.org/docs/gridsome-cli/) is installed globally and all the necessary npm modules are installed in the docs directory before testing in your local.**
- cd to _wp-content/themes/wp-theme-base/docs_
- Switch to the _docs_ branch
- Then switch to _ft-overview-section_ branch.
- On your terminal run the command **gridsome develop** to start the development server.
- Once the development server is running,  the site is available on **http://localhost:8080/** on your browser and should display the docs landing page.

## Screenshots
<!-- Add screenshots to show the problem and the solution; or delete the section entirely. -->

## Checklist
<!-- Replace  the [ ] with [x] to check the boxes. --> 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin
<!-- You must read and understand the following attestation. -->

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

JackieBinya commented:
I managed to retrieve the **urls** for the links which I had lost when I made the initial commit. Commit https://github.com/creativecommons/wp-theme-base/pull/30/commits/22c998c90b234ece26723fbc6eb366ebd8818b1a resolves that issue and updates the links.

hugosolar commented:
I added the login credentials for the staging website in our tracking document.
Can you please fix the conflict in this PR and commit again?

JackieBinya commented:
@hugosolar I have implemented the changes we discussed yesterday which were:
- Adopting the CC Vocabulary styling for the links: I deleted the default link styles in the style sheet which came with the theme.
- I took a screenshot from https://stage.creativecommons.org/  integrated it in the _docs_ homepage. The result is captured in the screenshots below:
**Screenshot-1**
![ONE-1](https://user-images.githubusercontent.com/50267279/93867358-39688a80-fcc9-11ea-9bf9-d43af1d455dd.png)

**Screenshot-2**
![two](https://user-images.githubusercontent.com/50267279/93867432-54d39580-fcc9-11ea-8887-a7b0bd184251.png)

In **Screenshot-2** the updated links are shown, my question is: Do we keep or trash the **heading** in **Screenshot-1**. I am feeling a little bit iffy about it, it looks a little bit dodgy visually. May I have your input on how I should proceed.
